lib/rspec-puppet/support.rb

```diff
@@ -228,9 +228,15 @@ def build_catalog(*args)
     end
 
     def munge_facts(facts)
-      output = {}
-      facts.keys.each { |key| output[key.to_s] = facts[key] }
-      output
+      return facts.reduce({}) do | memo, (k, v)|
```

Comment:
If you want to support strings and symbols as *keys* in the hash, please add test cases for both to show that it's working. especially if it needs to be special-cased within rspec-puppet.